build.sbt

```diff
@@ -69,5 +71,5 @@ libraryDependencies += "com.novocode" % "junit-interface" % "0.9" % "test"
 
 ScoverageSbtPlugin.ScoverageKeys.coverageHighlighting := {
   if (scalaBinaryVersion.value == "2.10") false
-  else false
+  else true
```

Comment:
This was an obvious typo. Changing this should give us more precise coverage metrics from Codecov.